GetData & HideData

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.19.2/axios.min.js"></script>
+    <script>
+    
+    
+    async function GetData(){
+      
+      var response= await axios('https://jsonplaceholder.typicode.com/users');
+      var data1= response.data;
+      
+      for(i=0;data1[i].id<=5;i++){
+        
+          document.getElementById('getData').innerHTML += "</br>Id: " +data1[i].id+
```

Comment:
use template string instead of concatenating

Reply:
template string?

Reply:
Google it and use that